dashboard/config/routes.rb

```diff
@@ -95,6 +95,15 @@ module OPS
     end
   end
 
+  # Used in react assessments tab
```

Comment:
I'd love feedback about these routes changes in particular. This works, but I haven't written a lot of this code so I'm not sure if this is a reasonable way to go about things.

Reply:
This looks great :)

Granted if it's only used in one place it doesn't need to be specified in a concern, though also there's no harm, just an extra level of indirection. Presumably we'll want to expose this in `/api/v1` as well, at which point the concern will be handy?